Shoko.Server/Settings/SettingsMigrations.cs

```diff
@@ -87,4 +88,26 @@ private static string MigrateAutoGroupRelationsAlternateToAlternative(string set
         var regex = new Regex(@"(?<=""AutoGroupSeriesRelationExclusions""\s*:\s*\[)(?<value>[^\]]+)", RegexOptions.Compiled);
         return regex.Replace(settings, match => match.Groups["value"].Value.Replace("alternate", "alternative", StringComparison.InvariantCultureIgnoreCase));
     }
+
+    private static string MigrateAniDBServerAddresses(string settings)
+    {
+        ushort serverPort = 9001;
+
+        var serverPortRegex = new Regex(@"(?<=""AniDb""\s*:\s*\{.*)""ServerPort""\s*:\s*(?<value>\d+)", RegexOptions.Compiled | RegexOptions.Singleline);
```

Comment:
These are fine to leave as is. Reusing compiled regexes helps speed up hot paths. This only runs once